content/docs/welcome-guide/setup/setup-from-github/building-linux.md

```diff
@@ -103,10 +103,11 @@ To prepare to build the engine and projects, choose one of the following build t
 1. Use CMake to build the engine as an SDK, the same as if you installed the engine from an installer tool. The following example shows the `profile` build configuration.
 
     ```shell
-    cmake --build build/linux --target INSTALL --config profile -j <number of parallel build tasks>
+    cmake --build build/linux --target INSTALL --config <debug|profile|release> -j <number of parallel build tasks>
     ```
 
     The `-j` is a recommended build tool optimization. It tells the Ninja build tool the number of parallel build tasks that will be executed simultaneously. We recommend that the 'number of parallel build tasks' matches the number of cores available on the Linux host machine.
+    The `--config` sets configuration type. This affects compilation flags. For example, `profile` turns on `-Werror`.
 
     Example:
 
```

Comment:
This comment is for line 119 (after "The engine takes a while to build..."), as I'm unable to make a comment there. 

In a "caution" callout box, we can mention the problem you encountered and a workaround. 
```suggestion

    {{< caution >}}
The `profile` configuration type enables the `-Werror` compilation flag. If you encounter build errors due to this, try building with the `debug` configuration instead. 
    {{< /caution >}}
    
```
----------------

Altogether, these suggested changes would render like this: 
![image](https://user-images.githubusercontent.com/75444793/152886013-4ba1af4b-02a9-4fdd-9fbe-3040b73b87b1.png)

Reply:
I have learned that every profile enables -Werror so this is not correct information.

See:
https://github.com/o3de/o3de/blob/development/cmake/Platform/Common/Clang/Configurations_clang.cmake

The problem I had was caused by git somehow getting commit where debug variable was unused in other profiles.